app/models/fs/api_connection.rb

```diff
@@ -56,7 +56,8 @@ def boxify
         api_connection: self,
         settings: {
           dic: subject["dic"],
-          subject_id: subject["subject_id"]
+          subject_id: subject["subject_id"],
+          message_drafts_import_enabled: Fs::Box::DISABLED_MESSAGE_DRAFTS_IMPORT_KEYWORDS.none? { |keyword| subject["name"].include?(keyword) }
```

Comment:
Imho by sme to mali aj na urovni admina umoznit zmenit cez gui, ale na skusanie staci takto.

Reply:
V prvej iteracii ako pisem aj hore to nie je prakticky ziaden zasah, lebo SPD formulare aj tak nepodporujeme. V dalsej, ked FS API bude podporovat aj SPD formulare, tak by sme vedeli priradenie draftu do schranky robit podla toho, aky typ formulara to je. Ak to je SPD formular, tak dame do SPD schranky, inak do hlavnej. Zakategorizovat to budeme vediet.